gapic/src/main/com/google/gapid/MainWindow.java

```diff
@@ -91,6 +92,7 @@
   protected final Map<MainTab.Type, Action> viewTabs = Maps.newHashMap();
   protected final Set<MainTab.Type> hiddenTabs = Sets.newHashSet();
   protected Action editCopy;
+  private StatusBar statusBar;
```

Comment:
I can't really find much by way of documentation for JFace's status bar. The little I can find seems like it's rather restrictive.

>  However, I've not added the status line yet, because if it's empty most of the time, which it looks like it would be in this change, it looks strange.

It looks stranger at the moment than it needs to - the bar appears to have padding that I have no idea how to remove. If you are really opposed to having a status bar, where would you suggest we put the notification?